src/qt/button.cpp

```diff
@@ -46,7 +46,14 @@ bool wxButton::Create(wxWindow *parent, wxWindowID id,
 {     
     QtCreate(parent);
     SetLabel( wxIsStockID( id ) ? wxGetStockLabel( id ) : label );
-
```

Comment:
This empty line could have been kept. In fact another could have been added at the end of the new block.
An `// apply label alignment` comment too?